tests/helpers/test_entity_platform.py

```diff
@@ -1392,3 +1392,44 @@ async def async_added_to_hass(self):
         """Make sure control is returned to the event loop on add."""
         await asyncio.sleep(0.1)
         await super().async_added_to_hass()
+
+
+@pytest.mark.parametrize(
+    "entity_name,expected_entity_id",
```

Comment:
Side note: It's possible to separate the parameters with space after the comma, which makes it easier to read in my opinion.